amt/site/templates/projects/new.html.j2

```diff
@@ -1,17 +1,18 @@
 {% extends 'layouts/base.html.j2' %}
 
 {% block content %}
-<div class="rvo-layout-column rvo-layout-gap--lg">
-    <div class="rvo-max-width-layout rvo-max-width-layout--md">
-        <main>
+
+    <div class="rvo-max-width-layout rvo-max-width-layout--md rvo-max-width-layout-inline-padding--none margin-bottom-large">
+
             <div class="rvo-form">
                 <div class="rvo-layout-column rvo-layout-gap--sm">
                     <div class="rvo-form-intro">
-                        <h1 class="utrecht-heading-1">{% trans %}Create new project{% endtrans %}</h1>
+                        <h1 class="utrecht-heading-1">{% trans %}Maak een nieuw project{% endtrans %}</h1>
```

Comment:
This does not seem to be translated in English. Also we use English as our base language in all the templates, so maybe change this to "Create a new project" and add translations accordingly.

Reply:
Hmm.. I thought we used Dutch as base language, so that is why this was changed.

Reply:
Ah ok, that could very well be the case. But then we might need to change all other trans tags in the Jinja template to Dutch to be consistent.

Reply:
My bad, it has to be English. I fixed all translations.